roles/ec2-app/tasks/destroy_elasticache.yml

```diff
@@ -1,13 +1,18 @@
 ---
+
+- name: Calculate identifier length
+  set_fact:
+    ecache_id_len: "{{  [ec_data.ec_name, opg_data.stack ] | join('-') | length  }}"
+
 - name: generate cluster identifier
   set_fact:
     ecc_instance_name: "{{ [ec_data.ec_name, opg_data.stack] | join('-') }}"
-  when: "{{ [ec_data.ec_name, opg_data.stack ] | join('-') | length <= 20}}"
+  when: ecache_id_len <= 20
 
 - name: generate cluster identifier
```

Comment:
I always find it difficult reading ansible output and what was done, so I was thinking  `generate cluster identifier long ecache` and `generate cluster identifier short ecache`